lib/rubygems/remote_fetcher.rb

```diff
@@ -92,6 +114,10 @@ def initialize(proxy=nil, dns=nil, headers={})
     @cert_files = Gem::Request.get_cert_files
 
     @headers = headers
+
+    @max_retries = max_retries.to_i() # Must be an int
```

Comment:
Not need to use parens at the end of `to_i`

Reply:
Okay, I changed it. I know it's not necessary, just stylistic choice. If possible, this should be added to rubocop I think, if this is rubygem's style convention.

Reply:
Agree. Thanks for making the changes and sorry for the hassle.

David just added a cop for this https://github.com/rubygems/rubygems/pull/3682